src/components/Pressable/GenericPressable/PropTypes.js

```diff
@@ -4,6 +4,19 @@ import CONST from '../../../CONST';
 
 const stylePropTypeWithFunction = PropTypes.oneOfType([stylePropType, PropTypes.func]);
 
+/**
+ * Custom test for required props
+ * + accessibilityLabel is required when accessible is true
+ * @param {Object} props
+ * @returns {Error} Error if prop is required
+ */
+function requiredPropsCheck(props) {
+    if (props.accessible !== true || (props.accessibilityLabel !== undefined && typeof props.accessibilityLabel === 'string')) {
+        return;
+    }
+    return new Error(`Provide a valid string for accessibilityLabel prop when accessible is true`);
+}
```

Comment:
```suggestion
function requiredPropsCheck(props) {
    if (props.accessible === true && (props.accessibilityLabel === undefined || typeof props.accessibilityLabel !== 'string')) {
        return new Error(`Provide a valid string for accessibilityLabel prop when accessible is true`);
    }
}
```

Makes this shorter and a little easier to understand (also follows the pattern of those other ones I found

Reply:
Well that's annoying. Okay, fine to keep as is!